backend/src/models/room.ts

```diff
@@ -3,6 +3,7 @@ import { PlayerState } from "./videoState";
 
 export default interface Room {
   name: string;
+  currVideoTitle: string;
```

Comment:
Hmm... would it make sense to just get rid of `currVideoTitle` and `currVideoId`, and add a new property `currVideo: Video` which has both the title and id

Reply:
I think I'll merge this for now so styling on the overall room can be worked on with the video title in place